pkg/cmd/projectCreate.go

```diff
@@ -489,9 +498,47 @@ func createTestProject(
 	return err
 }
 
+func createTestManager(
+	fileGenerator *utils.FileGenerator,
+	packageName string,
+  manager string,
+	forceOverwrite bool,
+	useMaven bool,
+	useGradle bool,
+	isDevelopment bool,
+) error {
+
+	targetFolderPath := packageName + "/" + packageName + "." + manager
+	log.Printf("Creating tests project %s\n", targetFolderPath)
+
+	// Create the base test folder
+	err := fileGenerator.CreateFolder(targetFolderPath)
+	if err == nil {
+    if useMaven {
+			err = createTestFolderPom(fileGenerator, targetFolderPath, packageName, manager, forceOverwrite)
+		}
+    
+    if useGradle {
+			err = createTestFolderGradle(fileGenerator, targetFolderPath, packageName, manager, forceOverwrite, isDevelopment)
+		}
+	}
+  
+  if err == nil {
+	  err = createJavaSourceFolder(fileGenerator, targetFolderPath, packageName, manager, forceOverwrite)
```

Comment:
This err is ignored. It gets immediately over-written by the next statement `err = createTestResourceFolder(fileGenerator, targetFolderPath, forceOverwrite)`

So if this error returned is non-nil then the method should return that error.

Reply:
Okay